src/main/java/org/jboss/xavier/analytics/pojo/output/workload/inventory/WorkloadInventoryReportModel.java

```diff
@@ -84,6 +88,7 @@
     private Boolean ssaEnabled;
     private Boolean insightsEnabled;
     private String osFamily;
+    private String category;
```

Comment:
The first time I quickly took a look at the code it was not easy to get what `category` means. This is just a suggestion you can skip this one if you want but could we change it to `categoryLevel`, `flagCategory`, or anything else more meaningful?

Reply:
As there's talk of possibly introducing a field for each flag's category at a later date, I'll rename this vmCategory

Reply:
Thanks @m-brophy